pydra/engine/core.py

```diff
@@ -558,7 +582,7 @@ def create_connections(self, task):
                 task.state = state.State(task.name, other_states=other_states)
 
     async def _run(self, submitter=None, **kwargs):
-        self.inputs = dc.replace(self.inputs, **kwargs)
+        # self.inputs = dc.replace(self.inputs, **kwargs) don't need it?
```

Comment:
same question as before.

Reply:
we do not allow running a workflow without a submitter, so I believe this can't be used